inst/include/dust/interface.hpp

```diff
@@ -40,6 +44,44 @@ cpp11::list dust_alloc(cpp11::list r_data, int step,
   return cpp11::writable::list({ptr, info});
 }
 
+// think about rng seeding here carefully; should accept either a raw
+// vector or an integer I think.
+template <typename T>
+cpp11::writable::doubles dust_simulate(cpp11::sexp r_steps,
```

Comment:
I'm not familiar with the cpp11 standard, but are copies the right thing here, or would a reference be better? (are they pointers already?)

Reply:
they're pointers already